src/devices/zigbeetlc.ts

```diff
@@ -0,0 +1,227 @@
+        cluster: 'hvacUserInterfaceCfg',
+        attribute: 'tempDisplayMode',
+        description: 'The unit of the temperature displayed on the device screen.',
+    }),
+};
+
+/*
+    ZigbeeTLc devices with the full supported option set:
+    - Temperature (+calibration)
+    - Humidity (+calibration)
+    - Display
+    - Comfort (Temperature / Humidity)
+*/
+const definitions = [
+    {
+        fingerprint: [
+            {modelID: 'LYWSD03MMC-z', manufacturerName: 'Xiaomi'},
+            {modelID: 'LYWSD03MMC-bz', manufacturerName: 'Xiaomi'},
+            {modelID: 'MHO-C122-z', manufacturerName: 'MiaoMiaoCe'},
+            {modelID: 'MHO-C122-bz', manufacturerName: 'MiaoMiaoCe'},
+            {modelID: 'MHO-C401N-z', manufacturerName: 'MiaoMiaoCe'},
+            {modelID: 'MHO-C401N-bz', manufacturerName: 'MiaoMiaoCe'},
+        ],
+        model: 'LYWSD03MMC',
+        vendor: 'Xiaomi',
+        description: 'Temp & RH Monitor Lite (pvxx/ZigbeeTLc)',
+        extend: [
+            temperature({reporting: {min: 10, max: 300, change: 10}}),
+            humidity({reporting: {min: 10, max: 300, change: 50}}),
+            extend.display,
+            extend.tempDisplayMode,
+            extend.comfortDisplay,
+            extend.comfortTemperatureMin,
+            extend.comfortTemperatureMax,
+            extend.comfortHumidityMin,
+            extend.comfortHumidityMax,
+            extend.tempCalibration,
+            extend.humidityCalibration,
+            extend.measurementInterval,
+            battery(),
+            extend.endpointQuirk,
+            ota(),
+        ],
+    },
+    /*
+        ZigbeeTLc devices supporting:
+        - Temperature (+calibration)
+        - Humidity (+calibration)
+        - Display
+    */
+    {
+        fingerprint: [
+            {modelID: 'CGDK2-z', manufacturerName: 'Qingping'},
+            {modelID: 'CGDK2-bz', manufacturerName: 'Qingping'},
+        ],
+        model: 'CGDK2',
+        vendor: 'Qingping',
+        description: 'Temp & RH Monitor Lite (pvxx/ZigbeeTLc)',
+        extend: [
+            temperature({reporting: {min: 10, max: 300, change: 10}}),
+            humidity({reporting: {min: 10, max: 300, change: 50}}),
+            extend.display,
+            extend.tempDisplayMode,
+            extend.tempCalibration,
+            extend.humidityCalibration,
+            extend.measurementInterval,
+            battery(),
+            extend.endpointQuirk,
+            ota(),
+        ],
+    },
+    /*
+        ZigbeeTLc devices supporting:
+        - Temperature (+calibration)
+        - Humidity (+calibration)
+    */
+    {
+        fingerprint: [
+            {modelID: 'TS0201-z', manufacturerName: 'Tuya'},
+            {modelID: 'TS0201-bz', manufacturerName: 'Tuya'},
+            {modelID: 'TH03Z-z', manufacturerName: 'Tuya'},
+            {modelID: 'TH03Z-bz', manufacturerName: 'Tuya'},
+        ],
+        model: 'WSD500A',
```

Comment:
```suggestion
        model: 'WSD500A_1',
```

`WSD500A` is already in use so I guess this makes the most sense.

Reply:
This a custom firmware for exactly those devices. The ZigbeeTLc firmware vastly improves the devices performance and reporting characteristics.

The same logic applies to the LYWSD03MMC model in the converter as well.

Keeping the above in mind, would you still like to have a model suffix for the same physical device but with different firmware? The Zigbee IDs are suffixed with (-z and -bz) for the custom firmware compared to the default already.

Reply:
Yes, because the `model` should be unique for every device.

Reply:
I've updated the duplicate models with custom ZigbeeTLc support to use an unique model name (suffix -z, same as the ZigbeeModel)